tab-jump-out

```diff
@@ -0,0 +1 @@
+(tab-jump-out :repo "cheunghy/tab-jump-out" :fetcher github)
```

Comment:
`tab-jump-out` is in top directory. You should add recipe file in `recipes` directory.